modules/nf-core/shapeit5/phaserare/meta.yml

```diff
@@ -0,0 +1,77 @@
+name: "shapeit5_phaserare"
+description: Tool to phase rare variants onto a scaffold of common variants (output of phase_common / ligate).
+keywords:
+  - phasing
+  - rare variants
+  - haplotype
+  - shapeit
+tools:
+  - "shapeit5":
+      description: "Fast and accurate method for estimation of haplotypes (phasing)"
+      homepage: "https://odelaneau.github.io/shapeit5/"
+      documentation: "https://odelaneau.github.io/shapeit5/docs/documentation"
+      tool_dev_url: "https://github.com/odelaneau/shapeit5"
+      doi: "10.1101/2022.10.19.512867 "
+      licence: "['MIT']"
+
+requirement: AVX2
```

Comment:
I don't think this will be shown in the help screen on the website. Can you add it to the main description too?

Reply:
Okay